synapse/models/inet.py

```diff
@@ -1539,6 +1611,85 @@ def getModelDefs(self):
                                 'doc': 'The host that the request was sent to.'}),
                         ),
                     }),
+
+                    ('inet:service:object', {
+
+                        'doc': 'Properties common to objects within a service platform.',
+                        'props': (
+
+                            ('id', ('str', {'strip': True}), {
+                                'doc': 'A platform specific ID used to identify the node.'}),
+
+                            ('created', ('time', {}), {
+                                'doc': 'The time when the object was created in the platform.'}),
+
+                            ('removed', ('time', {}), {
+                                'doc': 'The time when the object was removed from the platform.'}),
+
+                            ('status', ('inet:service:object:status', {}), {
+                                'doc': 'The status of this object.'}),
+
+                            ('platform', ('inet:service:platform', {}), {
+                                'doc': 'The platform which defines the object.'}),
+
+                            ('instance', ('inet:service:instance', {}), {
+                                'doc': 'The platform instance which defines the object.'}),
+
+                            ('creator', ('inet:service:account', {}), {
+                                'doc': 'The service account which created the object.'}),
+
+                            ('owner', ('inet:service:account', {}), {
+                                'doc': 'The serivce account which owns the object.'}),
+                        ),
+                    }),
+
+                    ('inet:service:action', {
```

Comment:
No specific comment (here) but noting to specifically sanity-check that all interface  -> form and interface -> interface -> forms make sense.
interfaces:
inet:service:action uses inet:service:object

forms:
it:host, it:host:tenancy, and it:software:image use inet:service:object
inet:service:login | message | access use inet:service:action